StardewArchipelago/ModEntry.cs

```diff
@@ -281,6 +283,12 @@ private void OnSaveLoaded(object sender, SaveLoadedEventArgs e)
                 _multiSleep.InjectMultiSleepOption(_archipelago.SlotData);
                 TravelingMerchantInjections.UpdateTravelingMerchantForToday(Game1.getLocationFromName("Forest") as Forest, Game1.dayOfMonth);
                 SeasonsRandomizer.ChangeMailKeysBasedOnSeasonsToDaysElapsed();
+                if (_archipelago.SlotData.Mods.HasMod(ModNames.SVE) && !Game1.player.mailReceived.Contains("GuntherUnlocked"))

```

Comment:
I don't want arbitrary mod code snippets in here. At **most**, one line for "DoModInitializationOnGameLoad()" and have the actual content in there.

Speaking of, I am just now noticing this weird "OnWarp" event being registered, then executing code that should not happen at each warp. It should probably also be moved out. And most important, not be static arrghh